doc/man/cockpit.conf.xml

```diff
@@ -76,6 +76,20 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
 <programlisting language="js">
 [WebService]
 ProtocolHeader = X-Forwarded-Proto
+</programlisting>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+        <varlistentry>
+          <term><option>ForwardedForHeader</option></term>
+          <listitem>
+            <para>Configure cockpit to look at the contents of this header to determine the real origin of a
+              connection.  This should only be used when cockpit is behind a reverse proxy, and care
+              should be taken to make sure that incoming requests cannot set this header.</para>
+          <informalexample>
+<programlisting language="js">
```

Comment:
Certainly not "js", more like "ini"? I know, "just" copy&pasta from the previous example, but let's fix it while we are at it.

Reply:
OK.

Reply:
Done.  Thanks for the catch.  Funny: "js" isn't even the right thing.  It's "javascript".